common-kafka-admin/src/main/java/com/cerner/common/kafka/admin/KafkaAdminClient.java

```diff
@@ -500,6 +513,8 @@ public void createTopic(String topic, int partitions, int replicationFactor, Pro
 
         if (!operationCompleted)
             throw new AdminOperationException("Timeout waiting for topic " + topic + " to be created");
+
+        updateTopicConfig(topic, topicConfig);
```

Comment:
Looks like you can supply the configuration when you create the topic [with NewTopic.configs(Map)](https://kafka.apache.org/20/javadoc/org/apache/kafka/clients/admin/NewTopic.html#configs-java.util.Map-)

Reply:
Updated: https://github.com/cerner/common-kafka/pull/16/commits/459298fe7ae187062df754bf280f8e3ed99220c7